bahmni-lite/.env

```diff
@@ -9,6 +9,8 @@ OPENMRS_HOST=openmrs
 OPENMRS_PORT=8080
 OPENMRS_ATOMFEED_USER=admin
 OPENMRS_ATOMFEED_PASSWORD=Admin123
+OPENMRS_SYSTEM_SMS_USER=systemuser
```

Comment:
Can we plz make this user as `system-user`.

Reply:
Sure will do the changes